plugin/log/README.md

```diff
@@ -72,6 +72,10 @@ The following place holders are supported:
 * `{>do}`: is the EDNS0 DO (DNSSEC OK) bit set in the query
 * `{>id}`: query ID
 * `{>opcode}`: query OPCODE
+* `{local}`: server's IP address, for IPv6 addresses these are enclosed in brackets: `[::1]`
+* `{A}`: number of the A type answers in the reply
+* `{AAAA}`: number of the AAAA type answers in the reply
+* `{resolver}`: the IP address and port of the proxy which replied to the query (used with the forward plugin only)
```

Comment:
I would propose:

* `{resolver}`: if query was forwarded to a resolver, the IP address and port of that resolver, else empty

Reply:
Ok. I only added "that replied resolver"